about.html

```diff
@@ -0,0 +1,129 @@
+            <a href="#"><p>Mypage</p></a>
+            <a href="#"><p>Korean</p></a>
+          </div>
+          <header>
+            <img class="hambuger" src="./assets/mobile//hamburger.png" alt="harmbuger icon"/>
+            <nav class="navigator toggle-ON">
+              <button class="exit-btn">X</button>
+              <a href="index.html"><img class="desktop" src="./assets/mobile/logo.svg" alt="logo"></a>
+              <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li class="desktop pdc"><a href="#">PD Campaign</a></li>
+              </ul>
+              <div>
+                <hr>
+              </div>
+            </nav>
+          </header>
+          <section class="main">
+            <div>
+            <h4 class="sharing">
+              "Hello! Sharing world"
+            </h4>
+            <h1>
+              POLISH YOUR SKILLS AND<span class="title-span"><br>GET READY</span>
+            </h1>
+            <hr>
+            </div>
+            <p class="support">            
+              The Global podcasting submit brings together the community of experts,
+              academics, activists who comprise the Creative commons affiliates network,
+              in a different country every two. This year, we hope to expand out invitation
+              list including organisations and individuals.<br><span class="desktop">So if your are active and engaged
+              in the worlds of open content and knowledge - Free software advocates, Wikipedias, open knowledge, galleries,
+              libraries, museums, archives, government and foundations - We hope you will conside joining us this year to build
+              a stronger more vibrant community.
+              </span>
+            </p>
+            <p>
+              Please contact us per Email for any further questions about PD Global Submit 2020!
+              <br><span class="mail"><a href="#">podcasting@pod.org</a></span>
+            </p>
+          </section>
+        </div>
+
+        <section class="programs">
+          <div class="programs-header">
+            <h2>
+              Creative Podcast Global Submit 2020 Logo
+            </h2>
+            <hr>
+          </div>
+          <p>
+            The logo of PD Global Submit 2020 was decided through the logo competition 
+            from 8 June to 9 July.<span class="desktop">After online public voting and selection
+            committee's judging process. the logo from Naresh Agrawal won the prize. After consultation
+            with Mr. Agrawal we optimized the price winning logo of the year's submit as follows 
+            </span>
+          </p>
+          <div class="logo-feature">
+            <img src="./assets/mobile/logo.svg" alt="logo">
+          </div>
```

Comment:
[optional] kindly fix the background colour. It doesn't match the one in your template

Reply:
Done.